src/test/java/io/confluent/kafkarest/mock/MockTime.java

```diff
@@ -25,11 +25,18 @@
 
   @Override
   public long milliseconds() {
+    // Incrementing currentMs prevents tests from locking up
+    // "waiting" for the time checks in the code.  Without this
+    // the ConsumerManager never completed (it never calls sleep or waitOn)
+    currentMs++;
```

Comment:
This shouldn't be necessary and it actually makes writing tests with MockTime a lot harder to reason about -- the point of MockTime is to let the test get some control over how time proceeds but now it is highly dependent on how many times `milliseconds()` is called. And the inconsistency between `milliseconds()` and `nanoseconds()` is confusing.

What was the test that got hung up when this wasn't incrementing? We probably just need to address the way that test works. The `ConsumerManager` may never directly wait/sleep, but something else should (e.g. consumer iterators).